kubernetes/team-riker/limit-range.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: 'v1'
+kind: 'LimitRange'
```

Comment:
what does this do?

Reply:
apiVersion - Which version of the Kubernetes API you're using to create this object - In this case v1 release of Kubernetes which has many core objects because it was the first stable released version.
kind - What kind of object you want to create. In this case, it is LimitRange which constrains the resource allocations.